src/main/groovy/io/openliberty/tools/gradle/tasks/AbstractServerTask.groovy

```diff
@@ -103,7 +102,7 @@ abstract class AbstractServerTask extends AbstractTask {
         } else if (project.liberty.outputDir != null) {
             return project.liberty.outputDir
         } else {
-            return getServerDir(project).toString()
+            return getUserDir(project).toString() + "/servers"
```

Comment:
Why would the server output go into the parent "servers" directory?

Reply:
Yeah, I guess this is kind of confusing. Going to rename this method to getOutputDir(). The ant tasks are deriving the serverOutputDir/serverDirectory from the outputDir we're getting here. I know we mention the default value of outputDir is `${installDir}/usr/servers/${serverName}` in the docs, which it clearly isn't here. It seems the default is coming from the serverDir value ant derives when the task calls init(). We don't explicitly set the value of the outputDir on the Gradle side of things. Since we call getServerOutputDir() when we generate the liberty-plugin-config.xml file for WDT I'll need to update this to use the same value as ant.